charts/nextcloud/values.yaml

```diff
@@ -508,6 +508,12 @@ metrics:
 
   # podLabels: {}
 
+  # nodeSelector: {}
```

Comment:
Want do we like, to use that without comment out?
```suggestion
  nodeSelector: {}
```

Reply:
I have no strong opinion, but in case we do this we could also drop the `with` part because we always know that there will be a value. Maybe a general consideration we should think about and then implement in the whole chart.

Reply:
Just let me know when you decided how to handle this in general and then I can update my pull requests so that it fits your requirements.

Reply:
Ah ok, I didn't know that. Then I prefer not commenting it out like you suggested.

Reply:
Ok I commented in all three properties but if this is suggested there are also a lot of other properties like podLabels or podAnnotations and they are still commented out? So not very consistent at the moment.